x-pack/plugins/threat_intelligence/cypress/integration/timeline.spec.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  BARCHART_TIMELINE_BUTTON,
+  FLYOUT_CLOSE_BUTTON,
+  FLYOUT_TABLE_ROW_TIMELINE_BUTTON,
+  INDICATOR_TYPE_CELL,
+  INDICATORS_TABLE_CELL_TIMELINE_BUTTON,
+  TIMELINE_DRAGGABLE_ITEM,
+  TOGGLE_FLYOUT_BUTTON,
+  UNTITLED_TIMELINE_BUTTON,
+} from '../screens/indicators';
+import { esArchiverLoad, esArchiverUnload } from '../tasks/es_archiver';
+import { login } from '../tasks/login';
+
+const THREAT_INTELLIGENCE_15Y_DATA =
+  '/app/security/threat_intelligence/indicators?indicators=(filterQuery:(language:kuery,query:%27%27),filters:!(),timeRange:(from:now-15y/d,to:now))';
+
+before(() => {
+  login();
+});
+
+describe('Indicators', () => {
+  before(() => {
+    esArchiverLoad('threat_intelligence');
+  });
+  after(() => {
+    esArchiverUnload('threat_intelligence');
+  });
+
+  describe('Indicators timeline interactions', () => {
+    before(() => {
+      cy.visit(THREAT_INTELLIGENCE_15Y_DATA);
```

Comment:
_Note to self:_ update this to what @lgmys has done on [his PR](https://github.com/elastic/kibana/pull/138882/files#diff-7b71364257ce94ff838bc343a1c864e7a6f93152f2c4ae3c5c28fade12904eadR31)